src/client/l10n/birthDateAndSsnFormats.ts

```diff
@@ -0,0 +1,23 @@
+import { MarketLabel } from './locales'
+
+export const ssnFormats: Record<MarketLabel, RegExp> = {
+  SE: /^((19|20))([0-9]{2})(0[1-9]|1[0-2])(0[1-9]|[1-2][0-9]|3[0-1])([0-9]{4})$/,
+  NO: /^(0[1-9]|[1-2][0-9]|3[0-1])(0[1-9]|1[0-2])([0-9]{2})([0-9]{5})$/,
+  DK: /^(0[1-9]|[1-2][0-9]|3[0-1])(0[1-9]|1[0-2])([0-9]{2})([0-9]{4})$/,
+}
+
+export const ssnLengths: Record<MarketLabel, number> = {
+  SE: 12,
+  NO: 11,
+  DK: 10,
+}
+
+const yearMonthDayFormat = /^(19[0-9]{2}|20[0-9]{2})-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])$/
+const dayMonthYearFormat = /^(0[1-9]|[1-2][0-9]|3[0-1])-(0[1-9]|1[0-2])-(19[0-9]{2}|20[0-9]{2})$/
+
+export const birthdateFormats: Record<MarketLabel | 'default', RegExp> = {
+  default: yearMonthDayFormat,
```

Comment:
How is default intended to be used? Don't we always know the market?

Reply:
Really good question, don't remember since I wrote that a long time ago (only moved it to a new file now) - will look into it 🙃

Reply:
OK so it seems to mostly be pretty poor naming form my part - it's the English format, which will be used for all English locales and I guess that could be thought of as some sort of "default". But maybe it's better to just call it `EN`? Or WDYT?

Reply:
Aha, yeah either that or add a "why" comment about it :)

Reply:
OK so actually it turned out it's the format we expect from back-end, and it's used for form validation in the details modal and to check that we have the expected format before we reverse in a function called `getFormattedBirthdate`.
So maybe a comment is enough, or call it something like `backEndDefault`? 🤔

Reply:
Yeah, anything that helps us remember!